### PR TITLE
Milestone 4

### PR DESCRIPTION
**Pull request by Israel-Laguan**
# Resume

I've followed the specifications as described in the project requirements, added the logic files and test suites for each file, and *came to the conclusion that I met all of the requirements*.

# Proposal

I want to have a review of the code to know if I need to make any changes. If no changes needed I want the project to be ACCEPTED so I can move to the next project with a happy smile :)

# Specifications

- Followed [these requirements](https://github.com/microverseinc/project-react-calculator/blob/master/milestones/MILESTONE_4.md) for Milestone 4
- Added folder `logic/`, `calculate.js` and `operate.js`
- Added tests for both calculate.js` and `operate.js`
- Checked `ESlint` & `Jest`
- Updated on Heroku

# [Live Version Here](https://math-magicians-calculator.herokuapp.com/)

# Tests

### `ESLint`
image

### `Jest`
image

### CODE REVIEW

File: src/logic/operate.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  if (!['+', '-', 'X', '÷'].includes(operation)) {
+    return numberOne.toString();
+  }
+  let operand = numberOne;
+  if (numberOne.toString().match('^\\.')) {
+    operand = `0${numberOne}`;
+  }
+  operand = Big(operand);
+  return {
+    '÷': numberTwo === '0' || numberTwo === 0 ? '0'
```
Comment (shloch) [inline src/logic/operate.js:21]: - why wrapped just `numberOne ` inside `big()`, u needed to also wrap `numberTwo`
- also concerning division, if `numberTwo == 0` we get INFINITY, not zero normally (you could also display an error if you like, something like `"division by ZERO error"`

**Review by shloch — APPROVED** [review]
## hello @Israel-Laguan 

Great job...You nailed the milestone, However take note of the comments below.
But you get a thumbs up though, because milestone 5 will make use of these files so, we'll se how their integration goes.

accomplished

### Happy coding

File: src/logic/operate.js (hunk 1 above)
Comment (Israel-Laguan) [reply src/logic/operate.js:21]: Thanks for your review! 

> * why wrapped just `numberOne` inside `big()`, u needed to also wrap `numberTwo`

About this I followed the documentation:
image

So it seems like the second number is optional to be inside `big()`. I would test if wrapping `numberTwo`  work as well!

> - also concerning division, if numberTwo == 0 we get INFINITY, not zero normally (you could also display an error if you like, something like "division by ZERO error"

I took care on that when I return '0' if second number is 0, look that is a ternary operator

**Comment by uptowndat777777** [comment]
Hello, Compliment of the day, how are you today ? i am Happy to get together with you,My name is Serach , I'm a military medical nurse from israel. Please let not my profession, race or nationality inch your mind, I have gone across your profile and became interested in you, we can be good friends, contact me through my private email address/  //abelmans@aol.com   ////

**Pull request merged**